app/src/main/java/intellistart/interviewplanning/controllers/dto/SlotDto.kt

```diff
@@ -0,0 +1,39 @@
+package intellistart.interviewplanning.controllers.dto
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.google.protobuf.Duration
+import com.google.type.Date
+import intellistart.interviewplanning.model.slot.Slot
+import java.time.LocalDate
+import intellistart.interviewplanning.commonmodels.slot.Slot as ProtobufSlot
+
+data class SlotDto(
+
+    val id: String = "",
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    val date: LocalDate = LocalDate.now(),
+
+    val from: String = "",
+
+    val to: String = ""
+)
+
+fun Slot.toDto(): SlotDto = SlotDto(
+    id = id.toHexString(),
+    date = period.date,
+    from = period.from.toString(),
+    to = period.to.toString()
+)
+
+fun Slot.toProto() = ProtobufSlot.newBuilder()
```

Comment:
[NIT] 

Given that this is a DTO class, you can create a separate class for mappers